node.go

```diff
@@ -109,9 +117,47 @@ func getProtocolParams(cfg *Config) *localstatequery.CurrentProtocolParamsResult
 	if err != nil {
 		return result
 	}
-	// Start our client
-	o.LocalStateQuery().Client.Start()
-	result, err = o.LocalStateQuery().Client.GetCurrentProtocolParams()
+	// Query our client
+	oConn.LocalStateQuery().Client.Start()
+	result, err = oConn.LocalStateQuery().Client.GetCurrentProtocolParams()
+	if err != nil {
+		return result
+	}
+	return result
+}
+
+// Get remote tip
+//
+//nolint:unused
+func getRemoteTip(cfg *Config, address string) *chainsync.Tip {
+	var result *chainsync.Tip
+	// Get a connection and setup our error channels
+	conn := createRemoteClientConnection(address)
+	if conn == nil {
+		return result
+	}
+	errorChan := make(chan error)
+	go func() {
+		for {
+			err := <-errorChan
+			fmt.Printf("ERROR: %s\n", err)
+			os.Exit(1)
+		}
+	}()
+	oConn, err := ouroboros.NewConnection(
+		ouroboros.WithConnection(conn),
+		ouroboros.WithNetworkMagic(uint32(cfg.Node.NetworkMagic)),
+		ouroboros.WithErrorChan(errorChan),
+		ouroboros.WithNodeToNode(true),
+		ouroboros.WithKeepAlive(false),
+		ouroboros.WithChainSyncConfig(buildChainSyncConfig()),
```

Comment:
It may still work without passing this at all if you're just getting the current tip

Reply:
I haven't tested that, yet. I'll let you know.